src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -174,7 +174,7 @@ PharoShortcuts >> delShortcut [
 { #category : #'keymaps - editor edit' }
 PharoShortcuts >> delWordShortcut [
 
-	^ Character delete shift asShortcut
+	^ Character delete ctrl asShortcut
```

Comment:
Good for Windows. Not sure about Mac support.

Reply:
> Control-D: Delete the character to the right of the insertion point. Or use Fn-Delete.
> Fn-Delete: Forward delete on keyboards that don't have a Forward Delete   key. Or use Control-D.

Not sure how to map this

Reply:
We can make it conditional for now.